src/GlobalStyle.js

```diff
@@ -8,21 +8,24 @@ const GlobalStyle = createGlobalStyle`
     font-family: Montserrat, sans-serif;
     font-weight: 700;
     font-size: 16px;
-    overflow-y: scroll;
   }
 
   html,
   body {
-    min-height: 100vh;
+    height: 100%;
+    width: 100%;
   }
 
   .page {
```

Comment:
This shouldn't be a global class. You can create a separate component `Page` or similar and use that as a wrapper component for your pages.